lib/yoti/dynamic_share_service/extension/thirdparty_attribute_extension.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Yoti
+  module DynamicSharingService
+    class ThirdPartyAttributeExtensionBuilder
+      def initialize
+        @expiry_date = nil
+        @definitions = []
+      end
+
+      def with_expiry_date(expiry_date)
+        @expiry_date = expiry_date
+        self
+      end
+
+      def with_definitions(*names)
+        names.each do |s|
+          @definitions += [{ name: s }]
```

Comment:
Providing an array to this method using `.with_definitions(['one', 'two'])` results in:
```
"definitions":[{"name":["one","two"]}]
```

Perhaps we should support `with_definitions` and `with_definition` (or rename to `with_definition`) to avoid confusion?

Reply:
.with_definitions('one', 'two') does what you expect as does .with_definitions(*['one','two'])

Reply:
ah, I that makes sense!